lib/mock-component/mock-component.ts

```diff
@@ -141,7 +143,9 @@ export function MockComponent<TComponent>(component: Type<TComponent>, metaData?
       this.__simulateTouch = fn;
     }
 
-    writeValue = (value: any) => {};
+    writeValue = (value: any) => {
```

Comment:
the idea of mocking is that you assert call of writeValue with proper args, `ControlValueAccessor` doesn't require to have `value` property.